libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/permission/PermissionsLauncherFragment.kt

```diff
@@ -13,20 +13,19 @@ import androidx.fragment.app.FragmentActivity
 /**
  * A view-less fragment that requests permissions using [ActivityResultCaller] interface.
  */
-internal class PermissionsLauncherFragment(
-    private val permissions: Array<String>,
-    private val onResult: ActivityResultCallback<Map<String, Boolean>>
-) : Fragment() {
+internal class PermissionsLauncherFragment : Fragment() {
 
+    private var permissions: Array<String>? = null
+    private var onResult: ActivityResultCallback<Map<String, Boolean>>? = null
```

Comment:
I could have used `PermissionsLauncherFragment.newInstance(args)` and `setArguments` but given the `internal` and `private` nature of this fragment, I found this simple enough.